# Description

It lists the most significant LV-trait associations for asthma, COPD and ACO.

# Modules

In [1]:
from pathlib import Path

import pandas as pd
import matplotlib.pyplot as plt
import matplotlib as mpl
import seaborn as sns
from IPython.display import HTML

import conf

# Settings

In [2]:
INPUT_DIR = conf.PROJECTS["ASTHMA_COPD"]["RESULTS_DIR"] / "gls_phenoplier"
display(INPUT_DIR)
assert INPUT_DIR.exists()

PosixPath('/opt/data/projects/asthma-copd/results/gls_phenoplier')

# Load data

## GLS associations for Asthma/COPD

In [3]:
input_filepath = INPUT_DIR / "gls-summary.pkl.gz"
assert input_filepath.exists()

In [4]:
gls_asthma_copd = pd.read_pickle(input_filepath)

In [5]:
gls_asthma_copd.shape

(2961, 6)

In [6]:
gls_asthma_copd.head()

lv      beta   beta_se        pvalue                         phenotype  \
0  LV247  0.667501  0.126759  7.200244e-08  GWAS_COPD_only_GLM_SNPs_info0.7-   
1  LV300  0.620979  0.125634  3.948175e-07  GWAS_COPD_only_GLM_SNPs_info0.7-   
2  LV180  0.590544  0.126656  1.592848e-06  GWAS_COPD_only_GLM_SNPs_info0.7-   
3  LV696  0.497310  0.125236  3.617882e-05  GWAS_COPD_only_GLM_SNPs_info0.7-   
4  LV504  0.478885  0.126907  8.120896e-05  GWAS_COPD_only_GLM_SNPs_info0.7-   

        fdr  
0  0.000107  
1  0.000234  
2  0.000619  
3  0.008927  
4  0.017176

## GLS associations for PhenomeXcan

In [7]:
input_filepath = INPUT_DIR / "gls-summary-phenomexcan.pkl.gz"
assert input_filepath.exists()

In [8]:
gls_phenomexcan = pd.read_pickle(input_filepath)

In [9]:
gls_phenomexcan.shape

(4037817, 5)

In [10]:
gls_phenomexcan.head()

phenotype        phenotype_desc     lv    pvalue       fdr
0  AB1_OTHER_VIRAL  Other viral diseases  LV736  0.004725  0.504339
1  AB1_OTHER_VIRAL  Other viral diseases  LV320  0.004848  0.508291
2  AB1_OTHER_VIRAL  Other viral diseases  LV366  0.005306  0.523691
3  AB1_OTHER_VIRAL  Other viral diseases  LV964  0.006106  0.548143
4  AB1_OTHER_VIRAL  Other viral diseases   LV92  0.006565  0.560048

## GLS associations for eMERGE

In [11]:
input_filepath = INPUT_DIR / "gls-summary-emerge.pkl.gz"
assert input_filepath.exists()

In [12]:
gls_emerge = pd.read_pickle(input_filepath)

In [13]:
gls_emerge.shape

(304983, 5)

In [14]:
gls_emerge.head()

phenotype  phenotype_desc     lv    pvalue       fdr
0  EUR_276.14  Hypopotassemia  LV273  0.000191  0.132187
1  EUR_276.14  Hypopotassemia   LV80  0.000616  0.244918
2  EUR_276.14  Hypopotassemia  LV870  0.000752  0.265545
3  EUR_276.14  Hypopotassemia  LV498  0.000914  0.293902
4  EUR_276.14  Hypopotassemia  LV561  0.001037  0.312601

# Top hits

In [15]:
with pd.option_context("display.max_columns", None, "display.max_colwidth", None):
    signif_lv_assocs = gls_asthma_copd.sort_values("fdr")  # .drop(columns="phenotype")
    signif_lv_assocs = signif_lv_assocs[signif_lv_assocs["fdr"] < 0.05]

    # convert back "category" data types into str/object
    signif_lv_assocs["lv"] = signif_lv_assocs["lv"].astype(str)
    signif_lv_assocs["phenotype"] = signif_lv_assocs["phenotype"].astype(str)

    display(signif_lv_assocs.shape)
    display(signif_lv_assocs.head(50))

(29, 6)

lv      beta   beta_se        pvalue  \
987   LV101  0.725541  0.126975  5.763470e-09   
0     LV247  0.667501  0.126759  7.200244e-08   
989    LV17  0.651945  0.130076  2.764702e-07   
988   LV444  0.626413  0.124808  2.666571e-07   
1     LV300  0.620979  0.125634  3.948175e-07   
1974  LV101  0.617122  0.128485  7.989790e-07   
2     LV180  0.590544  0.126656  1.592848e-06   
991    LV61  0.580480  0.125812  2.014374e-06   
990   LV948  0.581716  0.125679  1.876877e-06   
992   LV705  0.573511  0.124509  2.090083e-06   
1975  LV948  0.503273  0.126410  3.464972e-05   
3     LV696  0.497310  0.125236  3.617882e-05   
993    LV96  0.499062  0.127083  4.344623e-05   
4     LV504  0.478885  0.126907  8.120896e-05   
994    LV70  0.467698  0.125898  1.025075e-04   
995   LV844  0.491428  0.132480  1.047248e-04   
1976  LV705  0.466074  0.125974  1.088037e-04   
5      LV70  0.462430  0.127150  1.390485e-04   
996   LV207  0.465095  0.129773  1.705030e-04   
6     LV455  0.442878  0.124007  1.788404e-04   
1977  LV504  0.442245  0.126605  2.403098e-04   
8     LV799  0.437309  0.126230  2.674683e-04   
7     LV149  0.433460  0.124817  2.592485e-04   
997   LV506  0.436873  0.126588  2.809040e-04   
1978  LV563  0.441458  0.129181  3.181143e-04   
9     LV214  0.428885  0.125550  3.196640e-04   
998   LV140  0.443362  0.130312  3.361241e-04   
999   LV803  0.422627  0.125527  3.824215e-04   
10    LV383  0.420065  0.124709  3.803017e-04   

                               phenotype       fdr  
987   GWAS_Asthma_only_GLM_SNPs_info0.7-  0.000017  
0       GWAS_COPD_only_GLM_SNPs_info0.7-  0.000107  
989   GWAS_Asthma_only_GLM_SNPs_info0.7-  0.000205  
988   GWAS_Asthma_only_GLM_SNPs_info0.7-  0.000205  
1       GWAS_COPD_only_GLM_SNPs_info0.7-  0.000234  
1974          GWAS_ACO_GLM_SNPs_info0.7-  0.000394  
2       GWAS_COPD_only_GLM_SNPs_info0.7-  0.000619  
991   GWAS_Asthma_only_GLM_SNPs_info0.7-  0.000619  
990   GWAS_Asthma_only_GLM_SNPs_info0.7-  0.000619  
992   GWAS_Asthma_only_GLM_SNPs_info0.7-  0.000619  
1975          GWAS_ACO_GLM_SNPs_info0.7-  0.008927  
3       GWAS_COPD_only_GLM_SNPs_info0.7-  0.008927  
993   GWAS_Asthma_only_GLM_SNPs_info0.7-  0.009896  
4       GWAS_COPD_only_GLM_SNPs_info0.7-  0.017176  
994   GWAS_Asthma_only_GLM_SNPs_info0.7-  0.018951  
995   GWAS_Asthma_only_GLM_SNPs_info0.7-  0.018951  
1976          GWAS_ACO_GLM_SNPs_info0.7-  0.018951  
5       GWAS_COPD_only_GLM_SNPs_info0.7-  0.022873  
996   GWAS_Asthma_only_GLM_SNPs_info0.7-  0.026477  
6       GWAS_COPD_only_GLM_SNPs_info0.7-  0.026477  
1977          GWAS_ACO_GLM_SNPs_info0.7-  0.033884  
8       GWAS_COPD_only_GLM_SNPs_info0.7-  0.034434  
7       GWAS_COPD_only_GLM_SNPs_info0.7-  0.034434  
997   GWAS_Asthma_only_GLM_SNPs_info0.7-  0.034657  
1978          GWAS_ACO_GLM_SNPs_info0.7-  0.036405  
9       GWAS_COPD_only_GLM_SNPs_info0.7-  0.036405  
998   GWAS_Asthma_only_GLM_SNPs_info0.7-  0.036862  
999   GWAS_Asthma_only_GLM_SNPs_info0.7-  0.039047  
10      GWAS_COPD_only_GLM_SNPs_info0.7-  0.039047

## Hits per phenotype

In [16]:
signif_lv_assocs["phenotype"].value_counts()

GWAS_Asthma_only_GLM_SNPs_info0.7-    13
GWAS_COPD_only_GLM_SNPs_info0.7-      11
GWAS_ACO_GLM_SNPs_info0.7-             5
Name: phenotype, dtype: int64

In [17]:
signif_lv_assocs.groupby("phenotype")["fdr"].describe()

count      mean       std       min  \
phenotype                                                                 
GWAS_ACO_GLM_SNPs_info0.7-            5.0  0.019712  0.015569  0.000394   
GWAS_Asthma_only_GLM_SNPs_info0.7-   13.0  0.014394  0.015571  0.000017   
GWAS_COPD_only_GLM_SNPs_info0.7-     11.0  0.020067  0.015464  0.000107   

                                         25%       50%       75%       max  
phenotype                                                                   
GWAS_ACO_GLM_SNPs_info0.7-          0.008927  0.018951  0.033884  0.036405  
GWAS_Asthma_only_GLM_SNPs_info0.7-  0.000619  0.009896  0.026477  0.039047  
GWAS_COPD_only_GLM_SNPs_info0.7-    0.004773  0.022873  0.034434  0.039047

In [18]:
for idx, grp in signif_lv_assocs.groupby("phenotype"):
    display(HTML(f"<h3>{idx}</h3>"))
    display(grp)

lv      beta   beta_se        pvalue                   phenotype  \
1974  LV101  0.617122  0.128485  7.989790e-07  GWAS_ACO_GLM_SNPs_info0.7-   
1975  LV948  0.503273  0.126410  3.464972e-05  GWAS_ACO_GLM_SNPs_info0.7-   
1976  LV705  0.466074  0.125974  1.088037e-04  GWAS_ACO_GLM_SNPs_info0.7-   
1977  LV504  0.442245  0.126605  2.403098e-04  GWAS_ACO_GLM_SNPs_info0.7-   
1978  LV563  0.441458  0.129181  3.181143e-04  GWAS_ACO_GLM_SNPs_info0.7-   

           fdr  
1974  0.000394  
1975  0.008927  
1976  0.018951  
1977  0.033884  
1978  0.036405

lv      beta   beta_se        pvalue  \
987  LV101  0.725541  0.126975  5.763470e-09   
989   LV17  0.651945  0.130076  2.764702e-07   
988  LV444  0.626413  0.124808  2.666571e-07   
991   LV61  0.580480  0.125812  2.014374e-06   
990  LV948  0.581716  0.125679  1.876877e-06   
992  LV705  0.573511  0.124509  2.090083e-06   
993   LV96  0.499062  0.127083  4.344623e-05   
994   LV70  0.467698  0.125898  1.025075e-04   
995  LV844  0.491428  0.132480  1.047248e-04   
996  LV207  0.465095  0.129773  1.705030e-04   
997  LV506  0.436873  0.126588  2.809040e-04   
998  LV140  0.443362  0.130312  3.361241e-04   
999  LV803  0.422627  0.125527  3.824215e-04   

                              phenotype       fdr  
987  GWAS_Asthma_only_GLM_SNPs_info0.7-  0.000017  
989  GWAS_Asthma_only_GLM_SNPs_info0.7-  0.000205  
988  GWAS_Asthma_only_GLM_SNPs_info0.7-  0.000205  
991  GWAS_Asthma_only_GLM_SNPs_info0.7-  0.000619  
990  GWAS_Asthma_only_GLM_SNPs_info0.7-  0.000619  
992  GWAS_Asthma_only_GLM_SNPs_info0.7-  0.000619  
993  GWAS_Asthma_only_GLM_SNPs_info0.7-  0.009896  
994  GWAS_Asthma_only_GLM_SNPs_info0.7-  0.018951  
995  GWAS_Asthma_only_GLM_SNPs_info0.7-  0.018951  
996  GWAS_Asthma_only_GLM_SNPs_info0.7-  0.026477  
997  GWAS_Asthma_only_GLM_SNPs_info0.7-  0.034657  
998  GWAS_Asthma_only_GLM_SNPs_info0.7-  0.036862  
999  GWAS_Asthma_only_GLM_SNPs_info0.7-  0.039047

lv      beta   beta_se        pvalue                         phenotype  \
0   LV247  0.667501  0.126759  7.200244e-08  GWAS_COPD_only_GLM_SNPs_info0.7-   
1   LV300  0.620979  0.125634  3.948175e-07  GWAS_COPD_only_GLM_SNPs_info0.7-   
2   LV180  0.590544  0.126656  1.592848e-06  GWAS_COPD_only_GLM_SNPs_info0.7-   
3   LV696  0.497310  0.125236  3.617882e-05  GWAS_COPD_only_GLM_SNPs_info0.7-   
4   LV504  0.478885  0.126907  8.120896e-05  GWAS_COPD_only_GLM_SNPs_info0.7-   
5    LV70  0.462430  0.127150  1.390485e-04  GWAS_COPD_only_GLM_SNPs_info0.7-   
6   LV455  0.442878  0.124007  1.788404e-04  GWAS_COPD_only_GLM_SNPs_info0.7-   
8   LV799  0.437309  0.126230  2.674683e-04  GWAS_COPD_only_GLM_SNPs_info0.7-   
7   LV149  0.433460  0.124817  2.592485e-04  GWAS_COPD_only_GLM_SNPs_info0.7-   
9   LV214  0.428885  0.125550  3.196640e-04  GWAS_COPD_only_GLM_SNPs_info0.7-   
10  LV383  0.420065  0.124709  3.803017e-04  GWAS_COPD_only_GLM_SNPs_info0.7-   

         fdr  
0   0.000107  
1   0.000234  
2   0.000619  
3   0.008927  
4   0.017176  
5   0.022873  
6   0.026477  
8   0.034434  
7   0.034434  
9   0.036405  
10  0.039047

## Hits per LV

In [19]:
lv_assocs_by_size = signif_lv_assocs.groupby("lv").size().sort_values(ascending=False)
display(lv_assocs_by_size)

lv
LV101    2
LV70     2
LV948    2
LV705    2
LV504    2
LV207    1
LV61     1
LV149    1
LV844    1
LV803    1
LV799    1
LV17     1
LV180    1
LV696    1
LV563    1
LV214    1
LV506    1
LV140    1
LV455    1
LV444    1
LV383    1
LV300    1
LV247    1
LV96     1
dtype: int64

In [20]:
lv_assocs_by_size.shape

(24,)

# Take top LVs and show all associations

Take the top associated LVs (sorted by number of associations with asthma/COPD traits), and then show all the associations with the traits.

This allows to see whether one LV is strongly associated with some traits and not associated at all with the others.
For example, it could happen that one LVs is strongly associated with asthma and COPD using some threshold (pvalue/FDR < 0.05), but just slightly above that threshold for ACO (pvalue = 0.06). We would not trust much the "no association" with ACO in this case. However, if the p-value for ACO would be 0.76, we'd have more evidence that it is not associated, and thus the LV is more specific to asthma and COPD.

In [21]:
for lv_code in lv_assocs_by_size.index:
    display(HTML(f"<h2>{lv_code}</h2>"))

    with pd.option_context(
        "display.max_rows",
        None,
        "display.max_columns",
        None,
        "display.max_colwidth",
        None,
    ):
        display(HTML(f"<h3>Asthma/COPD/ACO</h3>"))
        _tmp = gls_asthma_copd[gls_asthma_copd["lv"] == lv_code].sort_values("pvalue")
        display(_tmp)

        display(HTML(f"<h3>PhenomeXcan</h3>"))
        _tmp = gls_phenomexcan[gls_phenomexcan["lv"] == lv_code].sort_values("pvalue")
        _tmp = _tmp[_tmp["fdr"] < 0.05]
        display(_tmp)

        display(HTML(f"<h3>eMERGE</h3>"))
        _tmp = gls_emerge[gls_emerge["lv"] == lv_code].sort_values("pvalue")
        _tmp = _tmp[_tmp["fdr"] < 0.20]
        display(_tmp)

lv      beta   beta_se        pvalue  \
987   LV101  0.725541  0.126975  5.763470e-09   
1974  LV101  0.617122  0.128485  7.989790e-07   
416   LV101 -0.008936  0.129442  5.275172e-01   

                               phenotype       fdr  
987   GWAS_Asthma_only_GLM_SNPs_info0.7-  0.000017  
1974          GWAS_ACO_GLM_SNPs_info0.7-  0.000394  
416     GWAS_COPD_only_GLM_SNPs_info0.7-  0.966732

phenotype  \
358281                                          6152_100   
2638251                                           6152_9   
1147881                                       20002_1111   
3565044                                            22127   
3893715                                           6152_8   
966273                                             22126   
1225854                                 20003_1140861998   
2060856                                        30210_raw   
1771666                                       20002_1387   
3777249                                        30120_raw   
3986494                                             2316   
4004259               Astle_et_al_2016_Lymphocyte_counts   
3140635                                            30150   
3997350               Astle_et_al_2016_Eosinophil_counts   
4002286  Astle_et_al_2016_Sum_eosinophil_basophil_counts   
2859339                                 20003_1140871266   
1047208                                       20002_1452   
526081                                         30180_raw   
1707511                                         3761_raw   
893236                                           129_raw   
3900625                                 20003_1140909702   
3547279                                 20003_1141176832   

                                                                                                                               phenotype_desc  \
358281                       Blood clot, DVT, bronchitis, emphysema, asthma, rhinitis, eczema, allergy diagnosed by doctor: None of the above   
2638251  Blood clot, DVT, bronchitis, emphysema, asthma, rhinitis, eczema, allergy diagnosed by doctor: Hayfever, allergic rhinitis or eczema   
1147881                                                                                        Non-cancer illness code, self-reported: asthma   
3565044                                                                                                               Doctor diagnosed asthma   
3893715                                 Blood clot, DVT, bronchitis, emphysema, asthma, rhinitis, eczema, allergy diagnosed by doctor: Asthma   
966273                                                                                         Doctor diagnosed hayfever or allergic rhinitis   
1225854                                                          Treatment/medication code: ventolin 100micrograms inhaler (20003_1140861998)   
2060856                                                                                                                Eosinophill percentage   
1771666                                                                    Non-cancer illness code, self-reported: hayfever/allergic rhinitis   
3777249                                                                                                                      Lymphocyte count   
3986494                                                                                         Wheeze or whistling in the chest in last year   
4004259                                                                                                                      Lymphocyte Count   
3140635                                                                                                                     Eosinophill count   
3997350                                                                                                                      Eosinophil Count   
4002286                                                                                                         Sum Eosinophil Basophil Count   
2859339                                                                        Treatment/medication code: arthrotec tablet (20003_1140871266)   
1047208                                                                             Non-cancer illness code, self-reported: eczema/dermatitis   
526081                                                  

Empty DataFrame
Columns: [phenotype, phenotype_desc, lv, pvalue, fdr]
Index: []

lv      beta   beta_se    pvalue                           phenotype  \
994   LV70  0.467698  0.125898  0.000103  GWAS_Asthma_only_GLM_SNPs_info0.7-   
5     LV70  0.462430  0.127150  0.000139    GWAS_COPD_only_GLM_SNPs_info0.7-   
2099  LV70  0.133317  0.127090  0.147109          GWAS_ACO_GLM_SNPs_info0.7-   

           fdr  
994   0.018951  
5     0.022873  
2099  0.966732

phenotype  \
1720341                                         20002_1464   
4007223    Astle_et_al_2016_Sum_basophil_neutrophil_counts   
3998340                  Astle_et_al_2016_Neutrophil_count   
4001301                 Astle_et_al_2016_Granulocyte_count   
4009197  Astle_et_al_2016_Sum_neutrophil_eosinophil_counts   
3999327          Astle_et_al_2016_Myeloid_white_cell_count   
1701588                                          30000_raw   
1116301                                   20003_1140869848   
3020221                                          30140_raw   
3996363            Astle_et_al_2016_White_blood_cell_count   
2159573                                               2887   
2845523                                         M13_RHEUMA   
1253492                                         RHEUMA_NOS   
933703                                  RHEUMA_SEROPOS_OTH   
724460                                                 M06   
1421282                                        STILL_ADULT   
3273881                                                M05   
319790                                      RHEUMA_SEROPOS   
2323398                                             1150_1   
2956065                                               2492   
85869                                              4803_14   
167806                                                2644   
3564066                                         20002_1377   
2928451                                               3476   
1875300                                           5087_raw   
2929436                                          30200_raw   
3042922                                            20107_3   
378027                                    20003_1140861958   
3062662                                               2188   

                                                         phenotype_desc    lv  \
1720341    Non-cancer illness code, self-reported: rheumatoid arthritis  LV70   
4007223                                   Sum Basophil Neutrophil Count  LV70   
3998340                                                Neutrophil Count  LV70   
4001301                                               Granulocyte Count  LV70   
4009197                                 Sum Neutrophil Eosinophil Count  LV70   
3999327                                        Myeloid White Cell Count  LV70   
1701588                              White blood cell (leukocyte) count  LV70   
1116301      Treatment/medication code: methotrexate (20003_1140869848)  LV70   
3020221                                               Neutrophill count  LV70   
3996363                                          White Blood Cell Count  LV70   
2159573                    Number of cigarettes previously smoked daily  LV70   
2845523                                            Rheumatoid arthritis  LV70   
1253492                          Other/unspecified rheumatoid arthritis  LV70   
933703               Other/unspecified seropositiverheumatoid arthritis  LV70   
724460           Diagnoses - main ICD10: M06 Other rheumatoid arthritis  LV70   
1421282                                       Adult-onset Still disease  LV70   
3273881   Diagnoses - main ICD10: M05 Seropositive rheumatoid arthritis  LV70   
319790                                Seropositive rheumatoid arthritis  LV70   
2323398                   Usual side of head for mobile phone use: Left  LV70   
2956065                           Taking other prescription medications  LV70   
85869                  Tinnitus: Yes, but not now, but have in the past  LV70   
167806                   Light smokers, at least 100 smokes in lifetime  LV70   
3564066  Non-cancer illness code, self-reported: polymyalgia rheumatica  LV70   
2928451                                Difficulty not smoking for 1 day  LV70   
1875300                                       Cylindrical power (right)  LV70   
2929436                                          Neutrophill percentage  LV70   
3

phenotype                   phenotype_desc    lv    pvalue       fdr
132258  EUR_496.1                        Emphysema  LV70  0.000030  0.042747
259581    EUR_288    Diseases of white blood cells  LV70  0.000034  0.045762
215169  EUR_288.2  Elevated white blood cell count  LV70  0.000183  0.129964

lv      beta   beta_se    pvalue                           phenotype  \
990   LV948  0.581716  0.125679  0.000002  GWAS_Asthma_only_GLM_SNPs_info0.7-   
1975  LV948  0.503273  0.126410  0.000035          GWAS_ACO_GLM_SNPs_info0.7-   
364   LV948  0.008905  0.127061  0.472063    GWAS_COPD_only_GLM_SNPs_info0.7-   

           fdr  
990   0.000619  
1975  0.008927  
364   0.966732

phenotype  \
358282                                          6152_100   
2638252                                           6152_9   
1771665                                       20002_1387   
966274                                             22126   
1147884                                       20002_1111   
3893718                                           6152_8   
3986495                                             2316   
1225856                                 20003_1140861998   
3565047                                            22127   
4011168                           IBD.EUR.Crohns_Disease   
2060859                                        30210_raw   
4015118               IBD.EUR.Inflammatory_Bowel_Disease   
1785483                                              K50   
1047210                                       20002_1452   
1453851                                         CHRONNAS   
3997355               Astle_et_al_2016_Eosinophil_counts   
3140640                                            30150   
1023520                                 20003_1140862772   
2391502                                       20002_1463   
4002290  Astle_et_al_2016_Sum_eosinophil_basophil_counts   

                                                                                                                               phenotype_desc  \
358282                       Blood clot, DVT, bronchitis, emphysema, asthma, rhinitis, eczema, allergy diagnosed by doctor: None of the above   
2638252  Blood clot, DVT, bronchitis, emphysema, asthma, rhinitis, eczema, allergy diagnosed by doctor: Hayfever, allergic rhinitis or eczema   
1771665                                                                    Non-cancer illness code, self-reported: hayfever/allergic rhinitis   
966274                                                                                         Doctor diagnosed hayfever or allergic rhinitis   
1147884                                                                                        Non-cancer illness code, self-reported: asthma   
3893718                                 Blood clot, DVT, bronchitis, emphysema, asthma, rhinitis, eczema, allergy diagnosed by doctor: Asthma   
3986495                                                                                         Wheeze or whistling in the chest in last year   
1225856                                                          Treatment/medication code: ventolin 100micrograms inhaler (20003_1140861998)   
3565047                                                                                                               Doctor diagnosed asthma   
4011168                                                                                                                        Crohns Disease   
2060859                                                                                                                Eosinophill percentage   
4015118                                                                                                            Inflammatory Bowel Disease   
1785483                                                                      Diagnoses - main ICD10: K50 Crohn's disease [regional enteritis]   
1047210                                                                             Non-cancer illness code, self-reported: eczema/dermatitis   
1453851                                                                                                                   Chron's disease NAS   
3997355                                                                                                                      Eosinophil Count   
3140640                                                                                                                     Eosinophill count   
1023520                                                                              Treatment/medication code: loratadine (20003_1140862772)   
2391502                        

Empty DataFrame
Columns: [phenotype, phenotype_desc, lv, pvalue, fdr]
Index: []

lv      beta   beta_se    pvalue                           phenotype  \
992   LV705  0.573511  0.124509  0.000002  GWAS_Asthma_only_GLM_SNPs_info0.7-   
1976  LV705  0.466074  0.125974  0.000109          GWAS_ACO_GLM_SNPs_info0.7-   
696   LV705 -0.087412  0.126804  0.754686    GWAS_COPD_only_GLM_SNPs_info0.7-   

           fdr  
992   0.000619  
1976  0.018951  
696   0.966732

phenotype  \
358283                                          6152_100   
2060858                                        30210_raw   
3893716                                           6152_8   
1147882                                       20002_1111   
2638253                                           6152_9   
3986496                                             2316   
3565045                                            22127   
966275                                             22126   
3140636                                            30150   
1225858                                 20003_1140861998   
1771668                                       20002_1387   
4015120               IBD.EUR.Inflammatory_Bowel_Disease   
3997352               Astle_et_al_2016_Eosinophil_counts   
1707512                                         3761_raw   
4002288  Astle_et_al_2016_Sum_eosinophil_basophil_counts   
4011176                           IBD.EUR.Crohns_Disease   

                                                                                                                               phenotype_desc  \
358283                       Blood clot, DVT, bronchitis, emphysema, asthma, rhinitis, eczema, allergy diagnosed by doctor: None of the above   
2060858                                                                                                                Eosinophill percentage   
3893716                                 Blood clot, DVT, bronchitis, emphysema, asthma, rhinitis, eczema, allergy diagnosed by doctor: Asthma   
1147882                                                                                        Non-cancer illness code, self-reported: asthma   
2638253  Blood clot, DVT, bronchitis, emphysema, asthma, rhinitis, eczema, allergy diagnosed by doctor: Hayfever, allergic rhinitis or eczema   
3986496                                                                                         Wheeze or whistling in the chest in last year   
3565045                                                                                                               Doctor diagnosed asthma   
966275                                                                                         Doctor diagnosed hayfever or allergic rhinitis   
3140636                                                                                                                     Eosinophill count   
1225858                                                          Treatment/medication code: ventolin 100micrograms inhaler (20003_1140861998)   
1771668                                                                    Non-cancer illness code, self-reported: hayfever/allergic rhinitis   
4015120                                                                                                            Inflammatory Bowel Disease   
3997352                                                                                                                      Eosinophil Count   
1707512                                                                                           Age hay fever, rhinitis or eczema diagnosed   
4002288                                                                                                         Sum Eosinophil Basophil Count   
4011176                                                                                                                        Crohns Disease   

            lv        pvalue           fdr  
358283   LV705  2.263189e-14  4.501646e-10  
2060858  LV705  9.915983e-11  8.187696e-07  
3893716  LV705  7.025707e-10  4.781029e-06  
1147882  LV705  1.132150e-09  7.314266e-06  
2638253  LV705  2.307144e-09  1.376045e-05  
3986496  LV705  4.837234e-09  2.661017e-05  
3565045  LV705  5.087978e-09  2.783784e-05  
966275   LV705  8.546386e-09  4.352927e-05  
3140636  LV705  2.826276e-08  1.270083e-04  
1225858  LV705  7.836944e-08  3.081222e-04  
1771668  LV705  1.293695e-07  4.731616e-04  
4015120  LV705  1.10

Empty DataFrame
Columns: [phenotype, phenotype_desc, lv, pvalue, fdr]
Index: []

lv      beta   beta_se    pvalue                           phenotype  \
4     LV504  0.478885  0.126907  0.000081    GWAS_COPD_only_GLM_SNPs_info0.7-   
1977  LV504  0.442245  0.126605  0.000240          GWAS_ACO_GLM_SNPs_info0.7-   
1124  LV504  0.120041  0.126935  0.172171  GWAS_Asthma_only_GLM_SNPs_info0.7-   

           fdr  
4     0.017176  
1977  0.033884  
1124  0.966732

phenotype  \
3489049            3456   
167797             2644   
2159584            2887   
80934    22601_21293047   
1603876      22617_2129   

                                                                  phenotype_desc  \
3489049  Number of cigarettes currently smoked daily (current cigarette smokers)   
167797                            Light smokers, at least 100 smokes in lifetime   
2159584                             Number of cigarettes previously smoked daily   
80934                Job coding: other engineering professional (22601_21293047)   
1603876                         Job SOC coding: Engineering professionals n.e.c.   

            lv        pvalue       fdr  
3489049  LV504  1.246404e-07  0.000458  
167797   LV504  4.594276e-07  0.001353  
2159584  LV504  2.338493e-06  0.005104  
80934    LV504  2.813422e-06  0.005914  
1603876  LV504  4.796321e-05  0.049104

phenotype  \
294126  EUR_440.22   
229974   EUR_440.2   

                                                                              phenotype_desc  \
294126  Atherosclerosis of native arteries of the extremities with intermittent claudication   
229974                                                    Atherosclerosis of the extremities   

           lv        pvalue       fdr  
294126  LV504  1.369999e-10  0.000003  
229974  LV504  1.534424e-06  0.004500

lv      beta   beta_se    pvalue                           phenotype  \
996   LV207  0.465095  0.129773  0.000171  GWAS_Asthma_only_GLM_SNPs_info0.7-   
1991  LV207  0.304300  0.130376  0.009813          GWAS_ACO_GLM_SNPs_info0.7-   
425   LV207 -0.013095  0.131125  0.539775    GWAS_COPD_only_GLM_SNPs_info0.7-   

           fdr  
996   0.026477  
1991  0.367787  
425   0.966732

phenotype  \
4007222    Astle_et_al_2016_Sum_basophil_neutrophil_counts   
3998339                  Astle_et_al_2016_Neutrophil_count   
2929416                                          30200_raw   
4001300                 Astle_et_al_2016_Granulocyte_count   
4009196  Astle_et_al_2016_Sum_neutrophil_eosinophil_counts   
1979925                                           3786_raw   
3999326          Astle_et_al_2016_Myeloid_white_cell_count   
526074                                           30180_raw   
3020220                                          30140_raw   
2125011                                          22147_raw   
1701591                                          30000_raw   
3996366            Astle_et_al_2016_White_blood_cell_count   
1001805                                               1309   
2661939                                               4979   

                                phenotype_desc     lv        pvalue  \
4007222          Sum Basophil Neutrophil Count  LV207  1.255312e-13   
3998339                       Neutrophil Count  LV207  1.402608e-13   
2929416                 Neutrophill percentage  LV207  4.759799e-13   
4001300                      Granulocyte Count  LV207  5.301957e-13   
4009196        Sum Neutrophil Eosinophil Count  LV207  5.973217e-13   
1979925                   Age asthma diagnosed  LV207  5.830995e-12   
3999326               Myeloid White Cell Count  LV207  1.086366e-11   
526074                   Lymphocyte percentage  LV207  3.622344e-10   
3020220                      Neutrophill count  LV207  6.676596e-10   
2125011         Age asthma diagnosed by doctor  LV207  2.817242e-08   
1701591     White blood cell (leukocyte) count  LV207  2.832558e-07   
3996366                 White Blood Cell Count  LV207  1.288481e-06   
1001805                     Fresh fruit intake  LV207  1.738935e-06   
2661939  FI5 : family relationship calculation  LV207  2.164401e-06   

                  fdr  
4007222  2.060456e-09  
3998339  2.256364e-09  
2929416  6.650241e-09  
4001300  7.356815e-09  
4009196  8.148229e-09  
1979925  6.052568e-08  
3999326  1.072505e-07  
526074   2.630641e-06  
3020220  4.561569e-06  
2125011  1.269478e-04  
1701591  9.084472e-04  
3996366  3.187899e-03  
1001805  4.033028e-03  
2661939  4.796626e-03

phenotype      phenotype_desc     lv    pvalue       fdr
107584  EUR_496.2  Chronic bronchitis  LV207  0.000134  0.113775

lv      beta   beta_se    pvalue                           phenotype  \
991   LV61  0.580480  0.125812  0.000002  GWAS_Asthma_only_GLM_SNPs_info0.7-   
2004  LV61  0.244032  0.125965  0.026376          GWAS_ACO_GLM_SNPs_info0.7-   
332   LV61  0.019247  0.126085  0.439339    GWAS_COPD_only_GLM_SNPs_info0.7-   

           fdr  
991   0.000619  
2004  0.600773  
332   0.966732

phenotype  \
2120077                                            30160   
1116302                                 20003_1140869848   
1253496                                       RHEUMA_NOS   
986020                                         30220_raw   
1023519                                 20003_1140862772   
1421285                                      STILL_ADULT   
3140639                                            30150   
724463                                               M06   
2845529                                       M13_RHEUMA   
2060862                                        30210_raw   
3986498                                             2316   
1395623                                        30070_raw   
290179                                  20003_1140870422   
1225859                                 20003_1140861998   
3893720                                           6152_8   
1147886                                       20002_1111   
2736951                                 20003_1140862952   
3974656                                        30280_raw   
3997356               Astle_et_al_2016_Eosinophil_counts   
3560109                                       22617_3229   
4002291  Astle_et_al_2016_Sum_eosinophil_basophil_counts   

                                                                                                phenotype_desc  \
2120077                                                                                        Basophill count   
1116302                                             Treatment/medication code: methotrexate (20003_1140869848)   
1253496                                                                 Other/unspecified rheumatoid arthritis   
986020                                                                                    Basophill percentage   
1023519                                               Treatment/medication code: loratadine (20003_1140862772)   
1421285                                                                              Adult-onset Still disease   
3140639                                                                                      Eosinophill count   
724463                                                  Diagnoses - main ICD10: M06 Other rheumatoid arthritis   
2845529                                                                                   Rheumatoid arthritis   
2060862                                                                                 Eosinophill percentage   
3986498                                                          Wheeze or whistling in the chest in last year   
1395623                                                        Red blood cell (erythrocyte) distribution width   
290179                                        Treatment/medication code: folic acid product (20003_1140870422)   
1225859                           Treatment/medication code: ventolin 100micrograms inhaler (20003_1140861998)   
3893720  Blood clot, DVT, bronchitis, emphysema, asthma, rhinitis, eczema, allergy diagnosed by doctor: Asthma   
1147886                                                         Non-cancer illness code, self-reported: asthma   
2736951                                   Treatment/medication code: mucodyne 375mg capsule (20003_1140862952)   
3974656                                                                         Immature reticulocyte fraction   
3997356                                                                                       Eosinophil Count   
3560109                                                                      Job SOC coding: Therapists n.e.c.   
4002291                                                                          Sum Eosinophil Basophil Count   

           lv        pvalue       fdr  
2120077  LV61  1.391367e-08  0.000067  
1116302  LV61  2.827777e-08  0.000127  
1253496  LV61  1.788095e-07  0.000620  
986020   LV61  2.233069e-07  0.000747  
1023519  LV61  2.

Empty DataFrame
Columns: [phenotype, phenotype_desc, lv, pvalue, fdr]
Index: []

lv      beta   beta_se    pvalue                           phenotype  \
7     LV149  0.433460  0.124817  0.000259    GWAS_COPD_only_GLM_SNPs_info0.7-   
2177  LV149  0.082753  0.124856  0.253746          GWAS_ACO_GLM_SNPs_info0.7-   
1200  LV149  0.066920  0.124877  0.296029  GWAS_Asthma_only_GLM_SNPs_info0.7-   

           fdr  
7     0.034434  
2177  0.966732  
1200  0.966732

phenotype  \
1966105               6152_5   
3533464                  I80   
139168            20002_1094   
2862304  I9_PHLETHROMBDVTLOW   
236882            20002_1445   
715579                I9_VTE   
2159564                 2887   
2202988        I9_DVTANDPULM   
2928439                 3476   
3489058                 3456   
692874            20002_1113   

                                                                                                                     phenotype_desc  \
1966105  Blood clot, DVT, bronchitis, emphysema, asthma, rhinitis, eczema, allergy diagnosed by doctor: Blood clot in the leg (DVT)   
3533464                                                                  Diagnoses - main ICD10: I80 Phlebitis and thrombophlebitis   
139168                                                         Non-cancer illness code, self-reported: deep venous thrombosis (dvt)   
2862304                                                                                                    DVT of lower extremities   
236882                                                 Non-cancer illness code, self-reported: clotting disorder/excessive bleeding   
715579                                                                                                       Venous thromboembolism   
2159564                                                                                Number of cigarettes previously smoked daily   
2202988                                                                             DVT of lower extremities and pulmonary embolism   
2928439                                                                                            Difficulty not smoking for 1 day   
3489058                                                     Number of cigarettes currently smoked daily (current cigarette smokers)   
692874                                                         Non-cancer illness code, self-reported: emphysema/chronic bronchitis   

            lv        pvalue           fdr  
1966105  LV149  3.118032e-14  6.082146e-10  
3533464  LV149  1.010187e-13  1.721076e-09  
139168   LV149  2.151560e-13  3.337343e-09  
2862304  LV149  2.988257e-11  2.705389e-07  
236882   LV149  3.220597e-09  1.881937e-05  
715579   LV149  4.824559e-09  2.657665e-05  
2159564  LV149  6.658500e-09  3.509896e-05  
2202988  LV149  7.213783e-09  3.763299e-05  
2928439  LV149  2.733096e-07  8.856936e-04  
3489058  LV149  1.065712e-06  2.718889e-03  
692874   LV149  2.312110e-05  2.933795e-02

phenotype                        phenotype_desc     lv        pvalue  \
26649     EUR_286                   Coagulation defects  LV149  1.035420e-17   
263532    EUR_452  Other venous embolism and thrombosis  LV149  1.884132e-06   
299066  EUR_452.2            Deep vein thrombosis [DVT]  LV149  1.925307e-04   

                 fdr  
26649   2.534221e-12  
263532  5.320631e-03  
299066  1.325476e-01

lv      beta   beta_se    pvalue                           phenotype  \
995   LV844  0.491428  0.132480  0.000105  GWAS_Asthma_only_GLM_SNPs_info0.7-   
1988  LV844  0.330251  0.132738  0.006436          GWAS_ACO_GLM_SNPs_info0.7-   
376   LV844  0.005067  0.133624  0.484876    GWAS_COPD_only_GLM_SNPs_info0.7-   

           fdr  
995   0.018951  
1988  0.276194  
376   0.966732

phenotype  \
3564058             20002_1377   
2975805             20002_1222   
143115              E4_DM1KETO   
3065622               2976_raw   
3409098                 1418_5   
1825950             20002_1502   
3366657        DM_KETOACIDOSIS   
4023999  RA_OKADA_TRANS_ETHNIC   
3124842           E4_DM1NOCOMP   
2131920                   2986   
3472267                 6177_3   
1046221       20003_1140883066   
1454839                 6153_3   
1619667                 E4_DM1   
2118103                    E10   
3900624       20003_1140909702   
3324216             20002_1456   
658329          22601_23133421   
319794          RHEUMA_SEROPOS   
1720348             20002_1464   
3021208              22146_raw   
933707      RHEUMA_SEROPOS_OTH   

                                                                                    phenotype_desc  \
3564058                             Non-cancer illness code, self-reported: polymyalgia rheumatica   
2975805                                    Non-cancer illness code, self-reported: type 1 diabetes   
143115                                                           Type 1 diabetes with ketoacidosis   
3065622                                                                     Age diabetes diagnosed   
3409098                                                         Milk type used: Other type of milk   
1825950                                       Non-cancer illness code, self-reported: appendicitis   
3366657                                                                      Diabetic ketoacidosis   
4023999                                                                       Rheumatoid Arthritis   
3124842                                                      Type 1 diabetes without complications   
2131920                                      Started insulin within one year diagnosis of diabetes   
3472267                            Medication for cholesterol, blood pressure or diabetes: Insulin   
1046221                              Treatment/medication code: insulin product (20003_1140883066)   
1454839  Medication for cholesterol, blood pressure, diabetes, or take exogenous hormones: Insulin   
1619667                                                                            Type 1 diabetes   
2118103                            Diagnoses - main ICD10: E10 Insulin-dependent diabetes mellitus   
3900624                                Treatment/medication code: sulfasalazine (20003_1140909702)   
3324216                      Non-cancer illness code, self-reported: malabsorption/coeliac disease   
658329                          Job coding: school inspector, education inspector (22601_23133421)   
319794                                                           Seropositive rheumatoid arthritis   
1720348                               Non-cancer illness code, self-reported: rheumatoid arthritis   
3021208                                      Age hayfever or allergic rhinitis diagnosed by doctor   
933707                                          Other/unspecified seropositiverheumatoid arthritis   

            lv        pvalue       fdr  
3564058  LV844  7.737268e-10  0.000005  
2975805  LV844  9.318352e-09  0.000047  
143115   LV844  2.245769e-08  0.000103  
3065622  LV844  1.017499e-07  0.000386  
3409098  LV844  1.551189e-07  0.000548  
1825950  LV844  1.758789e-07  0.000612  
3366657  LV844  2.118223e-07  0.000713  
4023999  LV844  2.227750e-07  0.000746  
3124842  LV844  3.438391e-07  0.001054  
2131920  LV844  4.376232e-07  0.001299  
3472267  LV844  1.514810e-06  0.003615  
1046221  LV844  2.556388e-06  0.005476  
1454839  LV844  4.085580e-06  0.007935  
1619667  LV844  5.817979e-06  0.010446  
2118103  LV844  6.032779e-06  0.010778  
3900624  LV844  6.202192e-06  0.011003  
3324216  LV844  2.522381e-05  0.031179  
658329   LV844  3.211954e-05  0.037140  
319794   LV844  3.392082e-05  0.038647  
1720348  LV844  3.831489e-05  0.042063  
3021208  LV844  4.141031e-05  

Empty DataFrame
Columns: [phenotype, phenotype_desc, lv, pvalue, fdr]
Index: []

lv      beta   beta_se    pvalue                           phenotype  \
999   LV803  0.422627  0.125527  0.000382  GWAS_Asthma_only_GLM_SNPs_info0.7-   
57    LV803  0.201791  0.125458  0.053894    GWAS_COPD_only_GLM_SNPs_info0.7-   
2583  LV803 -0.059228  0.125607  0.681362          GWAS_ACO_GLM_SNPs_info0.7-   

           fdr  
999   0.039047  
57    0.835501  
2583  0.966732

phenotype  \
1253491        RHEUMA_NOS   
4010187    MAGNETIC_HDL.C   
1116307  20003_1140869848   
2845528        M13_RHEUMA   
724465                M06   
1421287       STILL_ADULT   
3532474          5116_raw   
290182   20003_1140870422   
3982546  20003_1140872338   

                                                             phenotype_desc  \
1253491                              Other/unspecified rheumatoid arthritis   
4010187                                                 HDL Cholesterol NMR   
1116307          Treatment/medication code: methotrexate (20003_1140869848)   
2845528                                                Rheumatoid arthritis   
724465               Diagnoses - main ICD10: M06 Other rheumatoid arthritis   
1421287                                           Adult-onset Still disease   
3532474                                       3mm cylindrical power (right)   
290182     Treatment/medication code: folic acid product (20003_1140870422)   
3982546  Treatment/medication code: madopar 62.5 capsule (20003_1140872338)   

            lv        pvalue       fdr  
1253491  LV803  1.024053e-07  0.000388  
4010187  LV803  1.922145e-07  0.000658  
1116307  LV803  6.392289e-07  0.001792  
2845528  LV803  7.253431e-07  0.001982  
724465   LV803  1.637085e-06  0.003848  
1421287  LV803  1.643787e-06  0.003859  
3532474  LV803  1.496531e-05  0.021367  
290182   LV803  1.730340e-05  0.023743  
3982546  LV803  3.301767e-05  0.037903

Empty DataFrame
Columns: [phenotype, phenotype_desc, lv, pvalue, fdr]
Index: []

lv      beta   beta_se    pvalue                           phenotype  \
8     LV799  0.437309  0.126230  0.000267    GWAS_COPD_only_GLM_SNPs_info0.7-   
2021  LV799  0.216422  0.126223  0.043234          GWAS_ACO_GLM_SNPs_info0.7-   
1314  LV799  0.012331  0.126317  0.461119  GWAS_Asthma_only_GLM_SNPs_info0.7-   

           fdr  
8     0.034434  
2021  0.753027  
1314  0.966732

phenotype  \
2159563         2887   
3489050         3456   
2928442         3476   
167800          2644   
2534618         3466   
3042921      20107_3   
459942   I9_PERICARD   
2596797          I30   

                                                                  phenotype_desc  \
2159563                             Number of cigarettes previously smoked daily   
3489050  Number of cigarettes currently smoked daily (current cigarette smokers)   
2928442                                         Difficulty not smoking for 1 day   
167800                            Light smokers, at least 100 smokes in lifetime   
2534618                                      Time from waking to first cigarette   
3042921                                         Illnesses of father: Lung cancer   
459942                                                              Pericarditis   
2596797                           Diagnoses - main ICD10: I30 Acute pericarditis   

            lv        pvalue       fdr  
2159563  LV799  6.652650e-09  0.000035  
3489050  LV799  1.339144e-07  0.000486  
2928442  LV799  6.604166e-07  0.001834  
167800   LV799  1.100591e-06  0.002795  
2534618  LV799  2.659676e-06  0.005649  
3042921  LV799  3.518456e-06  0.007041  
459942   LV799  8.795384e-06  0.014332  
2596797  LV799  2.044226e-05  0.026661

Empty DataFrame
Columns: [phenotype, phenotype_desc, lv, pvalue, fdr]
Index: []

lv      beta   beta_se        pvalue  \
989   LV17  0.651945  0.130076  2.764702e-07   
2024  LV17  0.219086  0.131752  4.819318e-02   
323   LV17  0.022887  0.132530  4.314507e-01   

                               phenotype       fdr  
989   GWAS_Asthma_only_GLM_SNPs_info0.7-  0.000205  
2024          GWAS_ACO_GLM_SNPs_info0.7-  0.788398  
323     GWAS_COPD_only_GLM_SNPs_info0.7-  0.966732

phenotype  \
3986493                                             2316   
2060857                                        30210_raw   
3140634                                            30150   
1147883                                       20002_1111   
1116300                                 20003_1140869848   
3893717                                           6152_8   
358286                                          6152_100   
3565046                                            22127   
2952121                                        30190_raw   
1225857                                 20003_1140861998   
2845526                                       M13_RHEUMA   
1253499                                       RHEUMA_NOS   
1421286                                      STILL_ADULT   
724464                                               M06   
3997353               Astle_et_al_2016_Eosinophil_counts   
2340185                                        30130_raw   
1771669                                       20002_1387   
1737120                                          41218_1   
4002287  Astle_et_al_2016_Sum_eosinophil_basophil_counts   
2859340                                 20003_1140871266   
2929435                                        30200_raw   
2638266                                           6152_9   
966283                                             22126   
1286061                                            670_3   

                                                                                                                               phenotype_desc  \
3986493                                                                                         Wheeze or whistling in the chest in last year   
2060857                                                                                                                Eosinophill percentage   
3140634                                                                                                                     Eosinophill count   
1147883                                                                                        Non-cancer illness code, self-reported: asthma   
1116300                                                                            Treatment/medication code: methotrexate (20003_1140869848)   
3893717                                 Blood clot, DVT, bronchitis, emphysema, asthma, rhinitis, eczema, allergy diagnosed by doctor: Asthma   
358286                       Blood clot, DVT, bronchitis, emphysema, asthma, rhinitis, eczema, allergy diagnosed by doctor: None of the above   
3565046                                                                                                               Doctor diagnosed asthma   
2952121                                                                                                                   Monocyte percentage   
1225857                                                          Treatment/medication code: ventolin 100micrograms inhaler (20003_1140861998)   
2845526                                                                                                                  Rheumatoid arthritis   
1253499                                                                                                Other/unspecified rheumatoid arthritis   
1421286                                                                                                             Adult-onset Still disease   
724464                                                                                 Diagnoses - main ICD10: M06 Other rheumatoid arthritis   
3997353                                                                                                                      Eosinophil Count   
2340185                                                                                                                        Monocyte count   
1771669                                                                    Non-ca

phenotype                      phenotype_desc    lv    pvalue  \
90804   EUR_740.11  Osteoarthrosis, localized, primary  LV17  0.000038   
55273      EUR_244                      Hypothyroidism  LV17  0.000170   
252674   EUR_244.4                  Hypothyroidism NOS  LV17  0.000297   

             fdr  
90804   0.050087  
55273   0.125586  
252674  0.165183

lv      beta   beta_se    pvalue                           phenotype  \
2     LV180  0.590544  0.126656  0.000002    GWAS_COPD_only_GLM_SNPs_info0.7-   
2240  LV180  0.057438  0.126814  0.325307          GWAS_ACO_GLM_SNPs_info0.7-   
1745  LV180 -0.107228  0.126825  0.801061  GWAS_Asthma_only_GLM_SNPs_info0.7-   

           fdr  
2     0.000619  
2240  0.966732  
1745  0.966732

phenotype  \
2159565      2887   
2928447      3476   
3489072      3456   

                                                                  phenotype_desc  \
2159565                             Number of cigarettes previously smoked daily   
2928447                                         Difficulty not smoking for 1 day   
3489072  Number of cigarettes currently smoked daily (current cigarette smokers)   

            lv        pvalue       fdr  
2159565  LV180  1.310137e-08  0.000064  
2928447  LV180  4.634721e-06  0.008729  
3489072  LV180  1.173086e-05  0.017861

Empty DataFrame
Columns: [phenotype, phenotype_desc, lv, pvalue, fdr]
Index: []

lv      beta   beta_se    pvalue                           phenotype  \
3     LV696  0.497310  0.125236  0.000036    GWAS_COPD_only_GLM_SNPs_info0.7-   
2307  LV696  0.024192  0.125313  0.423462          GWAS_ACO_GLM_SNPs_info0.7-   
1302  LV696  0.018262  0.125336  0.442078  GWAS_Asthma_only_GLM_SNPs_info0.7-   

           fdr  
3     0.008927  
2307  0.966732  
1302  0.966732

phenotype                                phenotype_desc     lv  \
2928445      3476              Difficulty not smoking for 1 day  LV696   
2159591      2887  Number of cigarettes previously smoked daily  LV696   

           pvalue       fdr  
2928445  0.000003  0.006780  
2159591  0.000011  0.017166

phenotype                                    phenotype_desc     lv  \
229977  EUR_440.2                Atherosclerosis of the extremities  LV696   
231945    EUR_504  Other alveolar and parietoalveolar pneumonopathy  LV696   
279321    EUR_426                      Cardiac conduction disorders  LV696   

          pvalue       fdr  
229977  0.000007  0.014235  
231945  0.000147  0.115427  
279321  0.000192  0.132548

lv      beta   beta_se    pvalue                           phenotype  \
1978  LV563  0.441458  0.129181  0.000318          GWAS_ACO_GLM_SNPs_info0.7-   
1033  LV563  0.270752  0.129282  0.018137  GWAS_Asthma_only_GLM_SNPs_info0.7-   
377   LV563  0.004815  0.129388  0.485157    GWAS_COPD_only_GLM_SNPs_info0.7-   

           fdr  
1978  0.036405  
1033  0.492698  
377   0.966732

phenotype  \
1979926   3786_raw   
2430981  20015_raw   
1331463    20554_6   

                                                                             phenotype_desc  \
1979926                                                                Age asthma diagnosed   
2430981                                                                      Sitting height   
1331463  Actions taken following self-harm: Receive help from friends / family / neighbours   

            lv        pvalue       fdr  
1979926  LV563  2.216405e-09  0.000013  
2430981  LV563  4.975102e-06  0.009227  
1331463  LV563  9.942840e-06  0.015744

phenotype  \
177661   EUR_367   

                                                             phenotype_desc  \
177661  Disorders of refraction and accommodation; blindness and low vision   

           lv    pvalue       fdr  
177661  LV563  0.000013  0.022358

lv      beta   beta_se    pvalue                           phenotype  \
9     LV214  0.428885  0.125550  0.000320    GWAS_COPD_only_GLM_SNPs_info0.7-   
2233  LV214  0.058650  0.125584  0.320252          GWAS_ACO_GLM_SNPs_info0.7-   
1553  LV214 -0.054311  0.125608  0.667261  GWAS_Asthma_only_GLM_SNPs_info0.7-   

           fdr  
9     0.036405  
2233  0.966732  
1553  0.966732

phenotype  \
2159570      2887   
2928438      3476   
3489067      3456   
3834495    6157_2   

                                                                  phenotype_desc  \
2159570                             Number of cigarettes previously smoked daily   
2928438                                         Difficulty not smoking for 1 day   
3489067  Number of cigarettes currently smoked daily (current cigarette smokers)   
3834495                                     Why stopped smoking: Doctor's advice   

            lv        pvalue       fdr  
2159570  LV214  5.441465e-08  0.000224  
2928438  LV214  2.614890e-07  0.000856  
3489067  LV214  5.813166e-06  0.010445  
3834495  LV214  4.626319e-05  0.047886

phenotype                      phenotype_desc     lv    pvalue  \
163846   EUR_483  Acute bronchitis and bronchiolitis  LV214  0.000304   

             fdr  
163846  0.168022

lv      beta   beta_se    pvalue                           phenotype  \
997   LV506  0.436873  0.126588  0.000281  GWAS_Asthma_only_GLM_SNPs_info0.7-   
2070  LV506  0.157198  0.126727  0.107429          GWAS_ACO_GLM_SNPs_info0.7-   
489   LV506 -0.030562  0.126833  0.595205    GWAS_COPD_only_GLM_SNPs_info0.7-   

           fdr  
997   0.034657  
2070  0.966732  
489   0.966732

phenotype  \
3020225       30140_raw   
319793   RHEUMA_SEROPOS   
3273884             M05   
3564069      20002_1377   

                                                         phenotype_desc  \
3020225                                               Neutrophill count   
319793                                Seropositive rheumatoid arthritis   
3273884   Diagnoses - main ICD10: M05 Seropositive rheumatoid arthritis   
3564069  Non-cancer illness code, self-reported: polymyalgia rheumatica   

            lv    pvalue       fdr  
3020225  LV506  0.000005  0.009420  
319793   LV506  0.000015  0.021852  
3273884  LV506  0.000017  0.023658  
3564069  LV506  0.000037  0.041435

Empty DataFrame
Columns: [phenotype, phenotype_desc, lv, pvalue, fdr]
Index: []

lv      beta   beta_se    pvalue                           phenotype  \
998   LV140  0.443362  0.130312  0.000336  GWAS_Asthma_only_GLM_SNPs_info0.7-   
2269  LV140  0.048140  0.133140  0.358842          GWAS_ACO_GLM_SNPs_info0.7-   
974   LV140 -0.261595  0.133332  0.975096    GWAS_COPD_only_GLM_SNPs_info0.7-   

           fdr  
998   0.036862  
2269  0.966732  
974   0.981061

phenotype  \
3564057                          20002_1377   
276360                             6149_100   
986018                            30220_raw   
4015127  IBD.EUR.Inflammatory_Bowel_Disease   
1310739                          20002_1438   
1090635                              6141_4   
4035849          IBD.EUR.Ulcerative_Colitis   
1548603                          20001_1032   

                                                                    phenotype_desc  \
3564057             Non-cancer illness code, self-reported: polymyalgia rheumatica   
276360                              Mouth/teeth dental problems: None of the above   
986018                                                        Basophill percentage   
4015127                                                 Inflammatory Bowel Disease   
1310739                 Non-cancer illness code, self-reported: polycythaemia vera   
1090635   How are people in household related to participant: Mother and/or father   
4035849                                                         Ulcerative Colitis   
1548603  Cancer code, self-reported: brain cancer / primary malignant brain tumour   

            lv        pvalue           fdr  
3564057  LV140  5.958692e-12  6.169259e-08  
276360   LV140  5.186018e-08  2.149917e-04  
986018   LV140  1.032041e-07  3.905523e-04  
4015127  LV140  2.396801e-05  3.004608e-02  
1310739  LV140  2.628741e-05  3.216477e-02  
1090635  LV140  2.907992e-05  3.457579e-02  
4035849  LV140  3.157654e-05  3.665909e-02  
1548603  LV140  3.315409e-05  3.800597e-02

Empty DataFrame
Columns: [phenotype, phenotype_desc, lv, pvalue, fdr]
Index: []

lv      beta   beta_se    pvalue                           phenotype  \
6     LV455  0.442878  0.124007  0.000179    GWAS_COPD_only_GLM_SNPs_info0.7-   
2674  LV455 -0.086019  0.127093  0.750728          GWAS_ACO_GLM_SNPs_info0.7-   
1702  LV455 -0.096508  0.127129  0.776100  GWAS_Asthma_only_GLM_SNPs_info0.7-   

           fdr  
6     0.026477  
2674  0.966732  
1702  0.966732

phenotype  \
2159556      2887   
3489045      3456   
167790       2644   
2928433      3476   
1555512  20113_10   
1568343   20107_6   
2534616      3466   
195428    20110_6   

                                                                  phenotype_desc  \
2159556                             Number of cigarettes previously smoked daily   
3489045  Number of cigarettes currently smoked daily (current cigarette smokers)   
167790                            Light smokers, at least 100 smokes in lifetime   
2928433                                         Difficulty not smoking for 1 day   
1555512                Illnesses of adopted mother: Alzheimer's disease/dementia   
1568343                        Illnesses of father: Chronic bronchitis/emphysema   
2534616                                      Time from waking to first cigarette   
195428                         Illnesses of mother: Chronic bronchitis/emphysema   

            lv        pvalue           fdr  
2159556  LV455  4.843525e-16  1.303818e-11  
3489045  LV455  1.012036e-11  1.013999e-07  
167790   LV455  1.339894e-09  8.493321e-06  
2928433  LV455  1.224826e-07  4.508318e-04  
1555512  LV455  8.120929e-07  2.162983e-03  
1568343  LV455  1.184565e-06  2.968998e-03  
2534616  LV455  1.727509e-06  4.014124e-03  
195428   LV455  3.014378e-05  3.545443e-02

Empty DataFrame
Columns: [phenotype, phenotype_desc, lv, pvalue, fdr]
Index: []

lv      beta   beta_se        pvalue  \
988   LV444  0.626413  0.124808  2.666571e-07   
1982  LV444  0.339387  0.125579  3.449337e-03   
216   LV444  0.061175  0.125792  3.133789e-01   

                               phenotype       fdr  
988   GWAS_Asthma_only_GLM_SNPs_info0.7-  0.000205  
1982          GWAS_ACO_GLM_SNPs_info0.7-  0.192707  
216     GWAS_COPD_only_GLM_SNPs_info0.7-  0.966732

phenotype  \
358284                                          6152_100   
1225855                                 20003_1140861998   
1147885                                       20002_1111   
3893719                                           6152_8   
3986497                                             2316   
3547278                                 20003_1141176832   
3140637                                            30150   
2060860                                        30210_raw   
966279                                             22126   
3997354               Astle_et_al_2016_Eosinophil_counts   
2638260                                           6152_9   
4002289  Astle_et_al_2016_Sum_eosinophil_basophil_counts   
3024171                                             1498   
2066779                                 20003_1140881856   

                                                                                                                               phenotype_desc  \
358284                       Blood clot, DVT, bronchitis, emphysema, asthma, rhinitis, eczema, allergy diagnosed by doctor: None of the above   
1225855                                                          Treatment/medication code: ventolin 100micrograms inhaler (20003_1140861998)   
1147885                                                                                        Non-cancer illness code, self-reported: asthma   
3893719                                 Blood clot, DVT, bronchitis, emphysema, asthma, rhinitis, eczema, allergy diagnosed by doctor: Asthma   
3986497                                                                                         Wheeze or whistling in the chest in last year   
3547278                                                                    Treatment/medication code: seretide 50 evohaler (20003_1141176832)   
3140637                                                                                                                     Eosinophill count   
2060860                                                                                                                Eosinophill percentage   
966279                                                                                         Doctor diagnosed hayfever or allergic rhinitis   
3997354                                                                                                                      Eosinophil Count   
2638260  Blood clot, DVT, bronchitis, emphysema, asthma, rhinitis, eczema, allergy diagnosed by doctor: Hayfever, allergic rhinitis or eczema   
4002289                                                                                                         Sum Eosinophil Basophil Count   
3024171                                                                                                                         Coffee intake   
2066779                                                                              Treatment/medication code: salbutamol (20003_1140881856)   

            lv        pvalue           fdr  
358284   LV444  1.026069e-10  8.420894e-07  
1225855  LV444  4.236371e-09  2.352915e-05  
1147885  LV444  1.216036e-08  5.968315e-05  
3893719  LV444  1.773088e-08  8.344294e-05  
3986497  LV444  4.129243e-08  1.773737e-04  
3547278  LV444  1.314890e-07  4.800438e-04  
3140637  LV444  2.048920e-07  6.945429e-04  
2060860  LV444  2.335625e-07  7.742879e-04  
966279   LV444  2.119842e-06  4.706893e-03  
3997354  LV444  2.852419e-06  5.976932e-03  
2638260  LV444  1.220660e-05  1.836364e-02  
4002289  LV444  2.405283e-05  3.013370e-02  
3024171  LV444  2.622457e-05  3.212717e-02  
2066779  LV444  3.010526e-05  3.541944e-02

phenotype                           phenotype_desc     lv    pvalue  \
69091   EUR_472  Chronic pharyngitis and nasopharyngitis  LV444  0.000359   

            fdr  
69091  0.183788

lv      beta   beta_se    pvalue                           phenotype  \
10    LV383  0.420065  0.124709  0.000380    GWAS_COPD_only_GLM_SNPs_info0.7-   
2489  LV383 -0.031947  0.124742  0.601062          GWAS_ACO_GLM_SNPs_info0.7-   
1739  LV383 -0.104502  0.124750  0.798884  GWAS_Asthma_only_GLM_SNPs_info0.7-   

           fdr  
10    0.039047  
2489  0.966732  
1739  0.966732

phenotype  \
2159560      2887   
3489052      3456   
910034     1747_3   
2928448      3476   
2904741   20110_3   

                                                                  phenotype_desc  \
2159560                             Number of cigarettes previously smoked daily   
3489052  Number of cigarettes currently smoked daily (current cigarette smokers)   
910034                        Hair colour (natural, before greying): Light brown   
2928448                                         Difficulty not smoking for 1 day   
2904741                                         Illnesses of mother: Lung cancer   

            lv        pvalue       fdr  
2159560  LV383  4.159957e-09  0.000023  
3489052  LV383  2.774325e-07  0.000893  
910034   LV383  1.990519e-06  0.004483  
2928448  LV383  7.423163e-06  0.012589  
2904741  LV383  2.160032e-05  0.027802

phenotype           phenotype_desc     lv    pvalue       fdr
209244   EUR_572  Ascites (non malignant)  LV383  0.000413  0.199409

lv      beta   beta_se        pvalue  \
1     LV300  0.620979  0.125634  3.948175e-07   
1045  LV300  0.242374  0.125784  2.701813e-02   
2249  LV300  0.054606  0.125800  3.321265e-01   

                               phenotype       fdr  
1       GWAS_COPD_only_GLM_SNPs_info0.7-  0.000234  
1045  GWAS_Asthma_only_GLM_SNPs_info0.7-  0.604953  
2249          GWAS_ACO_GLM_SNPs_info0.7-  0.966732

phenotype  \
2159568                2887   
3273883                 M05   
1253503          RHEUMA_NOS   
319792       RHEUMA_SEROPOS   
167805                 2644   
2845532          M13_RHEUMA   
724469                  M06   
1421291         STILL_ADULT   
1116310    20003_1140869848   
2736952    20003_1140862952   
3489074                3456   
1047212          20002_1452   
933708   RHEUMA_SEROPOS_OTH   

                                                                  phenotype_desc  \
2159568                             Number of cigarettes previously smoked daily   
3273883            Diagnoses - main ICD10: M05 Seropositive rheumatoid arthritis   
1253503                                   Other/unspecified rheumatoid arthritis   
319792                                         Seropositive rheumatoid arthritis   
167805                            Light smokers, at least 100 smokes in lifetime   
2845532                                                     Rheumatoid arthritis   
724469                    Diagnoses - main ICD10: M06 Other rheumatoid arthritis   
1421291                                                Adult-onset Still disease   
1116310               Treatment/medication code: methotrexate (20003_1140869848)   
2736952     Treatment/medication code: mucodyne 375mg capsule (20003_1140862952)   
3489074  Number of cigarettes currently smoked daily (current cigarette smokers)   
1047212                Non-cancer illness code, self-reported: eczema/dermatitis   
933708                        Other/unspecified seropositiverheumatoid arthritis   

            lv        pvalue       fdr  
2159568  LV300  2.558877e-08  0.000116  
3273883  LV300  3.010026e-06  0.006255  
1253503  LV300  3.657624e-06  0.007257  
319792   LV300  3.685029e-06  0.007305  
167805   LV300  4.682618e-06  0.008790  
2845532  LV300  4.753141e-06  0.008892  
724469   LV300  7.813698e-06  0.013091  
1421291  LV300  9.421200e-06  0.015066  
1116310  LV300  1.589440e-05  0.022291  
2736952  LV300  1.949704e-05  0.025813  
3489074  LV300  2.708346e-05  0.032860  
1047212  LV300  3.353100e-05  0.038322  
933708   LV300  4.850086e-05  0.049491

Empty DataFrame
Columns: [phenotype, phenotype_desc, lv, pvalue, fdr]
Index: []

lv      beta   beta_se        pvalue  \
0     LV247  0.667501  0.126759  7.200244e-08   
2000  LV247  0.255201  0.126908  2.218842e-02   
1120  LV247  0.122543  0.126965  1.672472e-01   

                               phenotype       fdr  
0       GWAS_COPD_only_GLM_SNPs_info0.7-  0.000107  
2000          GWAS_ACO_GLM_SNPs_info0.7-  0.556779  
1120  GWAS_Asthma_only_GLM_SNPs_info0.7-  0.966732

phenotype  \
910022     1747_3   
2159575      2887   
167799       2644   
4016103  pgc.scz2   
3482140      2375   
3489066      3456   
1584146    1747_5   
2928452      3476   
3438715    1747_1   
3267958      1120   

                                                                  phenotype_desc  \
910022                        Hair colour (natural, before greying): Light brown   
2159575                             Number of cigarettes previously smoked daily   
167799                            Light smokers, at least 100 smokes in lifetime   
4016103                                                            Schizophrenia   
3482140                                        Relative age of first facial hair   
3489066  Number of cigarettes currently smoked daily (current cigarette smokers)   
1584146                             Hair colour (natural, before greying): Black   
2928452                                         Difficulty not smoking for 1 day   
3438715                            Hair colour (natural, before greying): Blonde   
3267958                            Weekly usage of mobile phone in last 3 months   

            lv        pvalue       fdr  
910022   LV247  2.915284e-07  0.000925  
2159575  LV247  3.157545e-07  0.000986  
167799   LV247  8.555324e-07  0.002259  
4016103  LV247  1.452552e-06  0.003495  
3482140  LV247  1.926975e-06  0.004366  
3489066  LV247  5.708801e-06  0.010286  
1584146  LV247  6.299140e-06  0.011121  
2928452  LV247  1.325515e-05  0.019512  
3438715  LV247  1.625560e-05  0.022618  
3267958  LV247  1.812479e-05  0.024509

phenotype                              phenotype_desc     lv  \
57246   EUR_721.1              Spondylosis without myelopathy  LV247   
129297    EUR_721            Spondylosis and allied disorders  LV247   
46390     EUR_509  Respiratory failure, insufficiency, arrest  LV247   

          pvalue       fdr  
57246   0.000087  0.085660  
129297  0.000088  0.085926  
46390   0.000182  0.129183

lv      beta   beta_se    pvalue                           phenotype  \
993   LV96  0.499062  0.127083  0.000043  GWAS_Asthma_only_GLM_SNPs_info0.7-   
1981  LV96  0.345562  0.126506  0.003160          GWAS_ACO_GLM_SNPs_info0.7-   
725   LV96 -0.098176  0.126988  0.780258    GWAS_COPD_only_GLM_SNPs_info0.7-   

           fdr  
993   0.009896  
1981  0.183474  
725   0.966732

phenotype  \
358285         6152_100   
2638255          6152_9   
1334424      40001_C679   
526083        30180_raw   
1771670      20002_1387   
3986499            2316   
1611771  22601_31152695   
966282            22126   
173712   22601_21283046   

                                                                                                                               phenotype_desc  \
358285                       Blood clot, DVT, bronchitis, emphysema, asthma, rhinitis, eczema, allergy diagnosed by doctor: None of the above   
2638255  Blood clot, DVT, bronchitis, emphysema, asthma, rhinitis, eczema, allergy diagnosed by doctor: Hayfever, allergic rhinitis or eczema   
1334424                                                                Underlying (primary) cause of death: ICD10: C67.9 Bladder, unspecified   
526083                                                                                                                  Lymphocyte percentage   
1771670                                                                    Non-cancer illness code, self-reported: hayfever/allergic rhinitis   
3986499                                                                                         Wheeze or whistling in the chest in last year   
1611771                                                                Job coding: quality assurance technician/co-ordinator (22601_31152695)   
966282                                                                                         Doctor diagnosed hayfever or allergic rhinitis   
173712                              Job coding: planning and quality control engineer, industrial engineer, quality engineer (22601_21283046)   

           lv        pvalue       fdr  
358285   LV96  3.061612e-10  0.000002  
2638255  LV96  3.567373e-08  0.000156  
1334424  LV96  8.003993e-08  0.000313  
526083   LV96  6.034138e-07  0.001706  
1771670  LV96  4.017653e-06  0.007832  
3986499  LV96  1.830985e-05  0.024685  
1611771  LV96  1.873009e-05  0.025117  
966282   LV96  2.751359e-05  0.033232  
173712   LV96  3.316026e-05  0.038006

Empty DataFrame
Columns: [phenotype, phenotype_desc, lv, pvalue, fdr]
Index: []

# Summary of results

* LV101:
  * asthma and ACO
  * very clear: tonsil innate lymphoid cells (ILC3), NK, ILC2
  * PhenomeXcan:
    * hayfever, allergic rhinitis, eczema, asthma, eosinophils
  * eMERGE:
    * None

* LV70:
  * asthma and COPD
  * Tissues:
    * It's very hard to find a single tissue for this LV
    * at the top are: blastocysts, keratinocytes, lung adenocarcinoma, etc
    * Human skeletal muscle myoblasts (RA)
      * there is a lot of variation here, but it is not possible to sepearate after reading the paper.
  * PhenomeXcan:
    * rheumatoid arthritis (RA), basophil/neutrophil count, neutrophil count, medication: methotrexate (cancer, RA)
  * eMERGE:
    * Emphysema (496.1), diseases of white blood cells (288)

* LV948:
  * asthma and ACO
  * very similar to LV101
  * PhenomeXcan:
    * very similar to LV101 + Cronh's disease + inflammatory bowel disease + ulcerative colitis
  * eMERGE:
    * None

* LV705:
  * asthma and ACO
  * very clear T cell signature when stimulated and also tonsil innate lymphoid cells (NK) (no ILC as in LV101)
  * PhenomeXcan:
    * very similar to LV948
  * eMERGE:
    * None

* LV504:
  * COPD and ACO
  * lung, but very weak signal
  * PhenomeXcan:
    * number of cigarettes currently smoked daily
  * eMERGE:
    * ahterosclerosis of the extremeties (440.*)

* LV207:
  * asthma
  * very strong breast cancer signal
  * PhenomeXcan:
    * neutrophil count and percentage, age of asthma diagnosed, white blood cell count
  * eMERGE:
    * None

* LV61:
  * asthma (ACO has FDR of 0.60 and unadjusted p-value of 0.026).
  * whole blood
  * PhenomeXcan:
    * basophil count, medication: methotrexate, RA, Adult-onset Still disease, asthma
  * eMERGE:
    * None

* LV149:
  * COPD (VERY INTERESTING!)
  * mostly expressed in embryonic kidney cell lines with single-site and multi-site mutations (something like that)
  * PhenomeXcan:
    * blood clot in the leg (DVT, deep venous thrombosis, number of cigarettes, emphysema
  * eMERGE:
    * Coagulation defects (286), other venous embolism and thrombosis (452)

* LV844:
  * asthma (ACO has FDR of 0.27 and unadjusted p-value of 0.006). 
  * TISSUES
  * PhenomeXcan:
    * polymyalgia rheumatica, type 1 diabetes, age diabetes diagnosed, RA, insulin medication, coeliac disease
  * eMERGE:
    * None

* LV803:
  * asthma (COPD has unadjusted p-value of 0.053)
  * tonsil (similar to LV101) + brain cell types
  * PhenomeXcan:
    * RA, HDL cholesterol, Adult-onset Still disease, keratometry measurements
  * eMERGE:
    * None

* LV799:
  * COPD (ACO unadjusted p-value of 0.043)
  * Blood, peripheral blood, multiple myeloma
  * PhenomeXcan:
    * cigarettes smoked, lung cancer (father), Pericarditis
  * eMERGE:
    * None

* LV17:
  * asthma (ACO has unadjusted p-value of 0.048, and COPD 0.43)
  * tonsil, simillar to LV101, but here NK are ate the top, and then ILC1 and ILC3, ILC2 comes very behind
  * PhenomeXcan:
    * wheeze, eosinophil, asthma, RA, hayfever/allergic rhinitis
  * eMERGE:
    * None

* LV180:
  * COPD (asthma and ACO pvalue > 0.30)
  * Glima + lung fibrobasts cells, hard to see
  * PhenomeXcan:
    * smoking, number of cigarettes
  * eMERGE:
    * None

* LV696:
  * COPD, rest has pvalue > 0.40
  * glioma + breast and lung cells, some tonsil ILC3 ILC2 ILC1, hard to see
  * PhenomeXcan:
    * smoking
  * eMERGE:
    * Atherosclerosis of the extremities

* LV563:
  * ACO (asthma has pvalue <0.02)
  * tonsil, but weird plot, with outliers, some astrocytes, fetal neocortex and other brain cell types
  * PhenomeXcan:
    * age asthma diagnosed, height
  * eMERGE:
    * Disorders of refraction and accommodation; blindness and low vision

* LV214:
  * COPD (rest has pvalue > 0.30)
  * Glioma + breast cancer, tonsil ILC3 NK ILC1 ILC2
  * PhenomeXcan:
    * smoking
  * eMERGE:
    * None

* LV506:
  * asthma (rest has pvalue > 0.10 for ACO, and < 0.50 for COPD)
  * Glioma + lung fibroblast cells, breast cancer, tonsil ILC3 NK ILC2
  * PhenomeXcan:
    * neutrophil count, RA, polymyalgia rheumatica
  * eMERGE:
    * None

* LV140:
  * asthma (rest pvalue > 0.35)
  * clear whole blood signature, with immune cells mostly or infected samples
  * PhenomeXcan:
    * polymyalgia rheumatica, dental problems, basophil percentage, IBD, polycythaemia vera, ulcerative colitis
  * eMERGE:
    * None

* LV455:
  * COPD (rest have pvalues > 0.75)
  * hard to see, HEK, brain cell types and cerebral organoids, glioma
  * PhenomeXcan:
    * smoking, chronic bronchitis/emphysema, Alzheimer's disease
  * eMERGE:
    * None

* LV444:
  * asthma and ACO (pvalue 0.003)
  * melanocytes, tonsil (ILC3, ILC2, ILC1), glioma + lung fibroblasts, brain cell types
  * PhenomeXcan:
    * asthma, eosinophils, hayfever
  * eMERGE:
    * None

* LV383:
  * COPD (not rest)
  * tonsil, glioma, brain cell types, hard to see
  * PhenomeXcan:
    * smoking, hair color/light brown, lung cancer
  * eMERGE:
    * None

* LV300:
  * COPD, asthma pvalue 0.027
  * glima + lung fibroblasts, tonsil (ILC3), cortex
  * PhenomeXcan:
    * cigarettes, RA, eczema/dermatitis
  * eMERGE:
    * None

* LV247:
  * COPD, ACO pvalue 0.022
  * skeletal muscle and skeletal muscle, lung fibroblasts, breeast cancer, glioma, tonsil (ILC3), brain cell types
  * PhenomeXcan:
    * hair color/light brown/black/blonde, cigarettes, schizophrenia, age first facial hair
  * eMERGE:
    * None

* LV96:
  * asthma and ACO (pvalue 0.003)
  * very clear tonsil signature: ILC2, ILC1, ILC3... NK to a very less extent.
  * PhenomeXcan:
    * asthma, hayfever/allergic rhinitis/eczema, lymphocyte percentage, wheeze
  * eMERGE:
    * ASSOC